Formatter/Pool.php

```diff
@@ -11,12 +11,14 @@
 
 namespace Sonata\FormatterBundle\Formatter;
 
+use Psr\Log\LoggerAwareInterface;
 use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
 use Twig_Environment;
 use Twig_Error_Syntax;
 use Twig_Sandbox_SecurityError;
 
-class Pool
+class Pool implements LoggerAwareInterface
```

Comment:
thats why i said per definition, but if we fullfill the interface, everything is fine.